pkg/bundle/sync_test.go

```diff
@@ -1522,6 +1605,25 @@ func Test_buildSourceBundle(t *testing.T) {
 				},
 			}
 
+			// for corresponding store if arbitrary password is expected then set it instead of default one
+			var password string
+			if test.expJKS {
+				if test.expPassword != nil {
+					password = *test.expPassword
+					test.bundle.Spec.Target.AdditionalFormats.JKS.Password = test.expPassword
+				} else {
+					password = DefaultJKSPassword
+				}
+			}
+			if test.expPKCS12 {
+				if test.expPassword != nil {
+					password = *test.expPassword
+					test.bundle.Spec.Target.AdditionalFormats.PKCS12.Password = test.expPassword
+				} else {
+					password = DefaultPKCS12Password
+				}
+			}
+
```

Comment:
If you apply the suggested changes above, I think this password logic could be simplified and added to the existing expJKS/expPKCS12 below. WDYT? I would prefer to not modify the bundle resource in the assert section.

Reply:
Ok, I agree with your suggestions

Reply:
Suggestions were implemented.